framework/include/distributions/DistributionBase.h

```diff
@@ -0,0 +1,45 @@
+/****************************************************************/
+/*               DO NOT MODIFY THIS HEADER                      */
+/* MOOSE - Multiphysics Object Oriented Simulation Environment  */
+/*                                                              */
+/*           (c) 2010 Battelle Energy Alliance, LLC             */
+/*                   ALL RIGHTS RESERVED                        */
+/*                                                              */
+/*          Prepared by Battelle Energy Alliance, LLC           */
+/*            Under Contract No. DE-AC07-05ID14517              */
+/*            With the U. S. Department of Energy               */
+/*                                                              */
+/*            See COPYRIGHT for full restrictions               */
+/****************************************************************/
+
+#ifndef DISTRIBUTIONBASE_H
+#define DISTRIBUTIONBASE_H
+
+#include "RandomGenerator.h"
+#include<string>
+
+const unsigned int _default_seed = 1031092845; 
```

Comment:
1. `_` is used for member variables not globals
2. Why is this in the header file?